src/force.cxx

```diff
@@ -532,10 +536,11 @@ void Force<TF>::create(Input& inputin, Netcdf_handle& input_nc, Stats<TF>& stats
         // Read the nudging profiles, which are the variable names with a "nudge" suffix
         for (auto& it : nudgelist)
         {
-            if (tdep_nudge.find(it) == tdep_nudge.end())
```

Comment:
is this going correctly if nudge is on, but a particular variable is not in the nudge list?